SurveyApp/Modules/Home/HomeInteractor.swift

```diff
@@ -49,15 +49,15 @@ extension HomeInteractor: HomeInteractorInput {
             switch result {
             case .success(let surveyResponse):
                 let surveys: [Survey] = surveyResponse.data.map {
+                    let attributes = $0.attributes as? SurveyAttributes
```

Comment:
We should remove any element whose `attributes` is `nil`, right? A survey without `title`, `description`, and `coverImageUrl` seem not a valid survey to display.

You can use [compactMap](https://developer.apple.com/documentation/swift/sequence/2950916-compactmap) here